### PR TITLE
Add mobile menu using javascript

### PR DESCRIPTION
**Pull request by AjroudiRida**
## Add mobile menu using javascript

in this project
- added a menu using the unordered list in the header
- added javascript to manage the click event and show or hide the menu.
- added the EsLinter to the repository

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const closeMenu = document.querySelector('#close-menu');
+hamburgerBtn.addEventListener('click', () => {
+  menu.classList.remove('hidden');
+  menu.classList.toggle('visible');
+});
+
+closeMenu.addEventListener('click', () => {
+  menu.classList.remove('visible');
+  menu.classList.toggle('hidden');
+});
```
Comment (danmainah) [inline index.js]: - [ ] @AjroudiRida and @hetrox8 great work adding the mobile menu, however,  some features are not yet implemented, and I am unable to close the menu by clicking any of the mobile menu options and to be taken to the correct part of the page depending on the option I choose. Kindly consider adding these features they are part of this milestone requirement as shown below
image

File: README.md [hunk 2]
```diff
@@ -154,7 +155,7 @@ Example command:
 
 ### Run tests
 
-To run tests, run the following command:  npx stylelint "**/*.{css,scss}".
+To run tests, run the following command:  npx stylelint "**/*.{css,scss}" and npx eslint .
```
Comment (danmainah) [inline README.md:158]: - [ ] [optional] @AjroudiRida kindly consider adding @hetrox8 as one of your authors , this makes the user to see all individuals who have contributed to this repo.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @AjroudiRida and @hetrox8 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Good design, it matches the Figma design given
✔️ Linter setup correctly and all checks are passing(except for webhint)


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by jerryowusu — APPROVED** [review]
Hi Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Great effort so far :+1: 
Close button works fine :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**